app/src/vuex/modules/account.js

```diff
@@ -31,47 +34,25 @@ export default ({ apollo }) => {
   const actions = {
     async listenToAuthChanges({ dispatch, commit }) {
       const Auth = (await getFirebase()).auth()
-      // start listening for idToken changes too
-      await dispatch(`listenToIdTokenChanges`)
       await new Promise((resolve) =>
         Auth.onAuthStateChanged(async (user) => {
           if (user) {
-            commit(`userSignedIn`, true)
+            // we only sign the user in, if we get a session from the API
+            // commit(`userSignedIn`, true)
             commit(`setUserInformation`, user)
 
-            const idToken = await user.getIdToken(/* forceRefresh */ true)
-            localStorage.setItem(`auth_token`, idToken)
             // make sure new authorization token get added to header
             apollo.cache.reset()
           } else {
-            localStorage.removeItem(`auth_token`)
-            commit(`userSignedIn`, false)
+            // user signed out
+            dispatch(`storeSession`, null)
             commit(`setUserInformation`, null)
           }
           resolve()
         })
       )
     },
-    async listenToIdTokenChanges({ commit }) {
```

Comment:
This wasn't working in the end? You said you were still getting outdated idTokens right

Reply:
This is not needed as we only send the idToken once for the sign in and we get it in line 65